src/core/Akka.Remote.Tests.MultiNode/Akka.Remote.Tests.MultiNode.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\common.props" />
+  <Import Project="..\..\xunitSettings.props" />
```

Comment:
Won't have any impact on the MNTR - those tests are already single threaded.

Reply:
Will remove this entries once CI will finish current run. It is going well so far

Reply:
Will remove this entries once CI will finish current run. It is going well so far